RELEASE_NOTES.rst

```diff
@@ -21,6 +21,114 @@
 
 .. towncrier release notes start
 
+Airflow 2.5.2 (2023-03-13)
```

Comment:
```suggestion
Airflow 2.5.2 (2023-03-14)
```

Or maybe even 15 now?

Reply:
Yes 15 is good, I'll follow up with the airflow-site update and the EndOfLife PR once the appropriate release date is set.

Fixed.

Reply:
I got additional instructions from Ephraim, this needs to be in a separate PR for easy cherry picking on v2-5-test.

Reply:
Oh, yes, good call.